src/EvaluationDetails.php

```diff
@@ -14,7 +14,7 @@ public function __construct(
         private readonly ?User $user,
         private readonly bool $isDefaultValue,
         private readonly ?string $error,
-        private readonly int $fetchTimeUnixSeconds,
+        private readonly float $fetchTimeUnixSeconds,
```

Comment:
At [ConfigCatClient.php:483](https://github.com/configcat/php-sdk/pull/36/commits/c1ec87685cf4d29a1abfb552f4b085c354e5a40c) looks like a msec value is passed but this field is second-based.